keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -423,8 +423,8 @@ def image_encoder(self):
         """
         if self._image_encoder is None:
             self._image_encoder = ImageEncoder(self.img_height, self.img_width)
-        if self.jit_compile:
-            self._image_encoder.compile(jit_compile=True)
+            if self.jit_compile:
```

Comment:
looks like a bad rebase in stable diffusion.

Reply:
Ohh, my word. Let me check that right away and get back to you on this.

Reply:
Please check now